packages/agrep/agrep.1.0/opam

```diff
@@ -5,6 +5,7 @@ homepage: "https://github.com/xavierleroy/ocamlagrep/"
 bug-reports: "https://github.com/xavierleroy/ocamlagrep/issues"
 dev-repo: "git+https://github.com/xavierleroy/ocamlagrep.git"
 license: "LGPL-2.0-or-later"
+x-maintenance-intent: [("latest")]
```

Comment:
```suggestion
x-maintenance-intent: ["(latest)"]
```

Sorry if this is documented differently somewhere. The quotes should be around the `(latest)`.

Reply:
That was cut and pasted from one of your posts on discuss.ocaml.org.  I suggest you edit your posts ASAP.

Reply:
Yes, thanks @xavierleroy. I already edited the post on discuss.